### PR TITLE
Making what "RIBs" is more clear on the README

### PR DESCRIPTION
**Pull request by gergelyorosz**
Reading some early feedback, it seems that it's not clear to some people meeting the architecture the first time what "RIBs" stands for. Sending a suggestion to make this clear, as well as linking the original announcement post (the latter we can update if we will have a more up-to-date one).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 [![Build Status](https://travis-ci.org/uber/RIBs.svg?branch=master)](https://travis-ci.org/uber/RIBs)
 
-RIBs is the cross-platform architecture behind many mobile apps at Uber. This architecture framework is designed for mobile apps with a large number of engineers and nested states.
+RIBs is the cross-platform architecture behind many mobile apps at Uber. The name RIBs is the short of Router, Interactor, Builder, which are core components of this architecture. This framework is designed for mobile apps with a large number of engineers and nested states.
```
Comment (sbarow) [inline README.md]: "The name RIBs is short" - remove "the"

File: README.md [hunk 2]
```diff
@@ -18,6 +18,8 @@ To get started with RIBs, please refer to the [RIBs documentation](https://githu
 
 To get more hands on with RIBs, we have written a [series of tutorials](https://github.com/uber/RIBs/wiki) that run you through the main aspects of the architecture with hands-on examples.
 
+To read about the backstory on why we created RIBs, see [this blog post](https://eng.uber.com/new-rider-app/) we wrote when releasing RIBs in production the first time.
```
Comment (sbarow) [inline README.md]: We also have an Uber mobility video on RIBs, not sure if the blog post references the video so might be nice to include a link also.

https://www.youtube.com/watch?v=Q5cTT0M0YXg

Comment (artman) [reply README.md]: Router, Interactor **AND** Builder?

File: README.md (hunk 1 above)
Comment (gergelyorosz) [reply README.md]: Thanks! Just did.

File: README.md (hunk 2 above)
Comment (gergelyorosz) [reply README.md]: Thanks, both! Addressed both of them.

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 
 [![Build Status](https://travis-ci.org/uber/RIBs.svg?branch=master)](https://travis-ci.org/uber/RIBs)
 
-RIBs is the cross-platform architecture behind many mobile apps at Uber. This architecture framework is designed for mobile apps with a large number of engineers and nested states.
+RIBs is the cross-platform architecture behind many mobile apps at Uber. The name RIBs is short of Router, Interactor and Builder, which are core components of this architecture. This framework is designed for mobile apps with a large number of engineers and nested states.
```
Comment (AttwellBrian) [inline README.md]: This is a good improvement.

File: README.md [hunk 4]
```diff
@@ -18,6 +18,8 @@ To get started with RIBs, please refer to the [RIBs documentation](https://githu
 
 To get more hands on with RIBs, we have written a [series of tutorials](https://github.com/uber/RIBs/wiki) that run you through the main aspects of the architecture with hands-on examples.
 
+To read about the backstory on why we created RIBs, see [this blog post](https://eng.uber.com/new-rider-app/) we wrote when releasing RIBs in production the first time and see [this short video](https://www.youtube.com/watch?v=Q5cTT0M0YXg) where we discussed how the RIBs architecture works.
```
Comment (AttwellBrian) [inline README.md]: I can't remember. Did yi's talk make sense without the tony's scoping talk coming first?

Comment (sbarow) [reply README.md]: Yeah I would say it makes sense - I watched it over the weekend, in saying that though I have a pretty good understanding of RIB's ;-)

**Comment by AttwellBrian** [comment]
👍

**Pull request merged**